github-graphql/PullRequestsOpen.gql

```diff
@@ -0,0 +1,26 @@
+query PullRequestsOpen ($repo_owner: String!, $repo_name: String!, $after: String) {
+  repository(owner: $repo_owner, name: $repo_name) {
+    pullRequests(first: 100, after: $after, states:OPEN, labels: ["S-waiting-on-review","T-compiler"]) {
+      pageInfo {
+        hasNextPage
+        endCursor
+      }
+      nodes {
+        number
+        updatedAt
+        createdAt
+        assignees(first: 10) {
+          nodes {
+            login
+            id
+          }
+        }
+        labels(first:5, orderBy:{field:NAME, direction:DESC}) {
```

Comment:
Looking at this, I assume you later use this to filter labels, but I don't think we should do that.

Reply:
right, currently I don't use them but I thought it could make sense to pull them anyway in the query (for future use?). Would it be ok to pull them but **not** use them for filtering?

Reply:
That's fine